websites/H/Heroku/presence.ts

```diff
@@ -0,0 +1,166 @@
+const presence = new Presence({
+		clientId: "1014999001801691246",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1e3);
```

Comment:
Why making something simple as complicated.
```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
```